configs/support/config.json

```diff
@@ -0,0 +1,23 @@
+{
+    "starport": {
+        "addr": "annex.montagu.dide.ic.ac.uk",
+        "ip": "129.31.26.49",
+        "user": "bb8",
+        "backup_location": "~bb8/starport/"
+    },
+    "targets": [
+        {
+          "name": "registry_data",
+          "type": "named_volume"
+        },
+        {
+            "type": "directory",
+            "path": "/montagu/vault/storage"
```

Comment:
Why do we not have vault using a volume?  Would be a really simple change in the vault setup and brings it further in line with the rest of the containerised bits

Reply:
Sure, lets open a ticket for this